pkg/docker/Dockerfile-xenial

```diff
@@ -9,7 +11,7 @@ RUN apt-get update \
     && apt-get install -f -y \
     && rm -rf /var/lib/apt/lists/* \
     && rm -rf /stackdriver-metadata.deb
+ADD entrypoint.sh /entrypoint.sh
```

Comment:
This is a local file — let's use `COPY`.

Reply:
Done.